modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -100,19 +99,16 @@ const loadIcons = (scriptLoader: ScriptLoader, editor: Editor, suffix: string) =
   });
 };
 
-const loadPlugins = (editor: Editor, settings: EditorSettings, suffix: string) => {
-  if (Type.isArray(settings.plugins)) {
-    settings.plugins = settings.plugins.join(' ');
-  }
-
-  Tools.each(settings.external_plugins, function (url, name) {
+const loadPlugins = (editor: Editor, suffix: string) => {
+  // These lines should be changed to a getParam and some type of setParam once such an API is available.
+  Tools.each(editor.getParam('external_plugins'), (url: string, name: string): void => {
     PluginManager.load(name, url, Fun.noop, undefined, () => {
       ErrorReporter.pluginLoadError(editor, url, name);
     });
-    settings.plugins += ' ' + name;
+    editor.settings.plugins += ' ' + name;
   });
 
-  Tools.each(settings.plugins.split(/[ ,]/), function (plugin) {
+  Tools.each(editor.settings.plugins.split(/[ ,]/), (plugin) => {
```

Comment:
This should be using `Settings.getPlugins(editor)`.

Reply:
Didn't we say not to do that for the plugins-value?

Reply:
We said don't do it in **plugins**, but it should always be done in **core**. Here's my previous comment:

> So to be clear, what you've got in the plugins is good. Core, however, should have this in Settings.ts.

Reply:
Right.